static/app/components/updatedEmptyState.tsx

```diff
@@ -75,11 +75,18 @@ export default function UpdatedEmptyState({project}: {project?: Project}) {
     replayOptions: {block: true, mask: true},
   };
 
+  if (currentPlatformKey === 'java' || currentPlatformKey === 'java-spring-boot') {
+    docParams.platformOptions = {
+      ...docParams.platformOptions,
+      packageManager: 'gradle',
```

Comment:
Is there a better place for this config to live?

Reply:
i was thinking about it and im don't think there is yet, typically this changes depending on what set up you want to do , but since this is issues focused and tailored for the empty states experience, with some exceptions for platforms, i think this might be the best place for now